style.css

```diff
@@ -0,0 +1,53 @@
+html {
+  font-size: 10px;
+}
+h1 {
+  font-size: 4rem;
+  font-family: Georgia, "Times New Roman", Times, serif;
+  letter-spacing: 0.5rem;
+}
+h4 {
+  margin: 50px;
+}
+a {
+  color: #32618a;
+}
+main {
+  background-color: rgb(240, 240, 240);

```

Comment:
Usually we use one system of colors in css (let's say, hex OR rgba), but not the mixture of them.

Reply:
Corrected!